### DISCUSSION TITLE
关于lua启用rime输入法的“时间戳”

### DISCUSSION THREAD

**Discussion by userplusuknow** [rime/home#1639] [Q&A] [answered]
*本人不是程序员 表述或者使用github会有一点差 谢谢体谅！

一、起因：
本人经常写日志 有个习惯是喜欢打时间戳 因此我想用rime实现的效果是：
输入 “ shijian ”  ，可以得到类似于“2024年08月08日 20:00” 这样的时间戳

二、目前的情况是：
1、win10系统
2、安装rime：0.16.1.0版本 （网址：https://rime.im/）
3、目前根据大佬的方案：https://github.com/rime/weasel/issues/63#issuecomment-956053259
发现自己的“用户文件夹”下面的文件，没有名字叫“luna_pinyin_simp.custom.yaml”的。
于是没办法继续了。

如图：
https://ibb.co/GssRq6p
https://ibb.co/dBx547C
https://ibb.co/ZzS6BNH
https://ibb.co/fHK2YpV

请教大佬的指点。

另外本人的rime 安装在D盘 每次卸载之后根据提示重启电脑 但是重装之后，总能在老地方发现我的备份文件夹（能发现它是因为我起名字比较怪：Backkkkkkkkup）  不知道这件事和我找不到“luna_pinyin_simp.custom.yaml”这个文件有关与否？

谢谢大佬抽空看我的提问！！

**Comment by rainorz** [comment, marked as answer]
首先 `luna_pinyin_simp.custom.yaml` 这个文件是“朙月拼音”的自定义配置
是需要自己创建的，内容用你看的文章里贴出来的就可以，如下：
```
patch:
  'engine/translators/+':
    - lua_translator@date_translator
```
然后如果你的输入方案不是“朙月拼音”，而是其他的，那应该创建输入法名称对应的文件。
如“五笔”，就对应的是`wubi86.custom.yaml`，再把上面相同的内容粘贴到这个文件里面。

最后确保同级目录的lua中添加了包含date_translator的xxx.lua脚本文件，再重新部署输入法就可以使用了。

**Reply by userplusuknow** [reply]
谢谢哥们儿！我不是程序员 所以之前一直没上网站 今天看到你的回答之后 照样子试了试 果然好了！
好人一生平安  再次感谢 祝福生活顺利 身体健康！
